docs/user/trusted-publishers/adding-a-publisher.md

```diff
@@ -97,7 +97,12 @@ each.
 
 === "ActiveState"
 
-    TODO
+    For ActiveState, you need to provide the project name on ActiveState's Platform,
+    the organization that project belongs to, and the username performing
+    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).
```

Comment:
I'm assuming this URL will start resolving sometime after we enable this feature?

Reply:
Yes, correct. It should be live soon. We'll get it set up so that it can be viewable before we go live.